.drone.star

```diff
@@ -19,8 +19,12 @@ config = {
 		},
 	},
 
+	'codestyle': True,
```

Comment:
Need to delete the `codestyle` section above this

Reply:
done!